master/internal/scheduler/fair_share_test.go

```diff
@@ -262,3 +262,85 @@ func TestFairShareLabels(t *testing.T) {
 	assertEqualToAllocate(t, toAllocate, expectedToAllocate)
 	assertEqualToRelease(t, taskList, toRelease, expectedToRelease)
 }
+
+func TestFairSharePreemptible(t *testing.T) {
+	agents := []*mockAgent{
+		{id: "agent", slots: 1, label: ""},
+	}
+	tasks := []*mockTask{
+		{id: "task1", slotsNeeded: 1, allocatedAgent: agents[0]},
+		{id: "task2", slotsNeeded: 1, allocatedAgent: agents[0]},
+	}
+
+	expectedToAllocate := []*mockTask{}
+	expectedToRelease := []*mockTask{tasks[1]}
+
+	system := actor.NewSystem(t.Name())
+	taskList, groupMap, agentMap := setupClusterStates(t, system, tasks, nil, agents)
+	toAllocate, toRelease := fairshareSchedule(taskList, groupMap, agentMap, BestFit)
+	assertEqualToAllocate(t, toAllocate, expectedToAllocate)
+	assertEqualToRelease(t, taskList, toRelease, expectedToRelease)
+}
+
+func TestFairShareHonorsCannotNonPreemptibleInAGroup(t *testing.T) {
```

Comment:
Should this be `TestFairShareHonorsNonPreemptibleInAGroup`?

Reply:
Yes!